docs/configuration_options.rst

```diff
@@ -96,6 +96,8 @@ Data Conversion
   * Importance: low
 
 ``topic.index.map``
+  This option is now deprecated. A future version may remove it completely. Please use single message transforms, such as RegexRouter, to map topic names to index names.
+
```

Comment:
The goal is that these .rst files can be largely generated by running `ElasticsearchSinkConnectorConfig.main(...)` and replacing nearly all of the file with the output (except the first few lines). To do that, we'd have to add this text to the documentation in the code.

Reply:
Thanks, @rhauch. Updated these lines using the main method.